### PR TITLE
tests for new endpoints

### PR DESCRIPTION
**Pull request by Nataliia-Popkova**
## PR Description

New tests for new endpoints

## Checklist
- [ x ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: resources/steps/service_point_steps.robot [hunk 1]
```diff
@@ -160,6 +160,44 @@ Create service type in DB
     END
     Disconnect From Database
 
+Create reagion in DB:
```
Comment (AntonShopin) [inline resources/steps/service_point_steps.robot:163]: Type, `region`

File: resources/steps/service_point_steps.robot [hunk 2]
```diff
@@ -160,6 +160,44 @@ Create service type in DB
...
+        ${name}=    Generate Random String    5    [LETTERS]
+    END
+    IF    '${uuid}' == '${None}'
+        ${uuid}=    Generate Random String    5    [LETTERS]
+    END
+    Connect to Spryker DB
+    IF    '${db_engine}' == 'pymysql'
+        Execute Sql String    insert ignore into spy_region (fk_country, iso2_code, name, uuid) value ('${country}', '${iso2_code}', '${name}','${uuid}');
+    ELSE
+        Execute Sql String    INSERT INTO spy_region (id_region, fk_country, iso2_code, name, uuid) VALUES (1, '${country}', '${iso2_code}', '${name}','${uuid}');
```
Comment (AntonShopin) [inline resources/steps/service_point_steps.robot:180]: This part is flaky. mariaDB has autoincrement but postgres - no. 
It's possible that id_region = 1 is already in the table and insert will fail. 

First you need to check 'next possible id' to insert. 

Example
`${new_id}=    Get next id from table    spy_region    id_region`

File: resources/steps/service_point_steps.robot [hunk 3]
```diff
@@ -160,6 +160,44 @@ Create service type in DB
...
+    [Arguments]    ${key}
+    Connect to Spryker DB
+    IF    '${db_engine}' == 'pymysql' 
+    ${servicePointUuid}    Query    SELECT uuid FROM spy_service_point WHERE `key` = '${key}' ORDER BY uuid DESC LIMIT 1;
+    ELSE
+    ${servicePointUuid}    Query    SELECT uuid FROM spy_service_point WHERE "key" = '${key}' ORDER BY uuid DESC LIMIT 1;
+    END
+    Disconnect From Database
+    Set Test Variable    ${servicePointUuid}     ${servicePointUuid[0][0]}
+    [Return]    ${servicePointUuid[0][0]}
```
Comment (AntonShopin) [inline resources/steps/service_point_steps.robot:199]: is this part correct? 
You create a variable `${servicePointUuid}` and add `${servicePointUuid[0][0]}` as a value. 
Then you return from this keyword variable value, not `${servicePointUuid}`

File: resources/steps/warehouse_user_assigment_steps.robot [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        ...    *Example:*
+        ...
+        ...    ``Get_warehouse_user_assigment_id:   ${warehouse[0].warehouse_uuid}    ${warehous_user[0].user_uuid}`
+        ...
+    [Arguments]    ${warehouse_uuid}    ${user_uuid}
+    Connect to Spryker DB
+    IF    '${db_engine}' == 'pymysql'
+    ${id_warehouse_user_assigment}=   Query    SELECT id_warehouse_user_assignment FROM spy_warehouse_user_assignment WHERE uuid = '${warehouse[0].warehouse_uuid}' AND user_uuid='${warehous_user[0].user_uuid}';
+    ELSE
+       ${id_warehouse_user_assigment}=    Query    SELECT id_warehouse_user_assignment FROM spy_warehouse_user_assignment WHERE uuid = '${warehouse[0].warehouse_uuid}' AND user_uuid='${warehous_user[0].user_uuid}';
```
Comment (AntonShopin) [inline resources/steps/warehouse_user_assigment_steps.robot:36]: I didn't find the difference between these 2 queries. Do we need `IF` here?

File: resources/steps/warehouse_user_assigment_steps.robot [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        ...    ``Remove_warehous_user_assigment:    ${warehouse[0].warehouse_uuid}    ${warehous_user[0].user_uuid}``
+        ...
+    [Arguments]    ${warehouse[0].warehouse_uuid}    ${warehous_user[0].user_uuid}
+        ${id_warehouse_user_assigment}=    Get_warehouse_user_assigment_id:   ${warehouse[0].warehouse_uuid}    ${warehous_user[0].user_uuid} 
+        Connect to Spryker DB
+    IF    '${db_engine}' == 'pymysql' 
+    Execute Sql String    DELETE FROM spy_warehouse_user_assignment WHERE id_warehouse_user_assignment = ${id_warehouse_user_assigment};
+    ELSE
+    Execute Sql String    DELETE FROM spy_warehouse_user_assignment WHERE id_warehouse_user_assignment = ${id_warehouse_user_assigment};
+    END
```
Comment (AntonShopin) [inline resources/steps/warehouse_user_assigment_steps.robot:55]: do we need 'IF' here?

File: tests/api/suite/bapi/service_points/service_point_addresses/nagative.robot [hunk 6]
```diff
@@ -0,0 +1,175 @@
+*** Settings ***
```
Comment (AntonShopin) [inline tests/api/suite/bapi/service_points/service_point_addresses/nagative.robot:1]: typo in the file name, nagative vs negative

File: tests/ui/regression/b2c.robot [hunk 7]
```diff
@@ -153,6 +153,7 @@ Catalog_Actions
     Yves: 1st product card in catalog (not)contains:     Add to Cart    false
     Yves: perform search by:    002
     Yves: 1st product card in catalog (not)contains:      Add to Cart    true
+    Set Timeout    2s
```
Comment (AntonShopin) [inline tests/ui/regression/b2c.robot:156]: please remove this line

**Review by AntonShopin — CHANGES_REQUESTED** [review]
For all keywords that retun variable, please add in the documentation section variable name that is in the return, so we don't need each time to check the code to get what variable is returned

File: resources/steps/service_point_steps.robot (hunk 1 above)
Comment (Nataliia-Popkova) [reply resources/steps/service_point_steps.robot]: fixed

File: resources/steps/service_point_steps.robot (hunk 2 above)
Comment (Nataliia-Popkova) [reply resources/steps/service_point_steps.robot]: fixed

File: resources/steps/service_point_steps.robot (hunk 3 above)
Comment (Nataliia-Popkova) [reply resources/steps/service_point_steps.robot]: fixed

File: resources/steps/warehouse_user_assigment_steps.robot (hunk 4 above)
Comment (Nataliia-Popkova) [reply resources/steps/warehouse_user_assigment_steps.robot]: fixed

File: resources/steps/warehouse_user_assigment_steps.robot (hunk 5 above)
Comment (Nataliia-Popkova) [reply resources/steps/warehouse_user_assigment_steps.robot]: fixed

File: tests/api/suite/bapi/service_points/service_point_addresses/nagative.robot (hunk 6 above)
Comment (Nataliia-Popkova) [reply tests/api/suite/bapi/service_points/service_point_addresses/nagative.robot]: fixed

File: tests/ui/regression/b2c.robot (hunk 7 above)
Comment (Nataliia-Popkova) [reply tests/ui/regression/b2c.robot]: removed

**Review by AntonShopin — APPROVED** [review]

**Pull request merged**